sbin/root-mount

```diff
@@ -136,6 +139,15 @@ factory_reset_if_needed()
 		if factory-reset-lvm $LVM_ROOT_SIZE $LVM_RESERVED_MB; then
 			# Successful recovery.
 
+			if test "$CLEAR_EXTERNAL_MEDIA" = "1"; then
+				CLEAR_EXTERNAL_COMMAND=`which factory-reset-external`
```

Comment:
Why do you need "which" here? Either you able to execute command by looking in $PATH (which is equally to calling "which") or not.